static/js/settings_muted_users.js

```diff
@@ -0,0 +1,24 @@
+import $ from "jquery";
+
+import * as muting_ui from "./muting_ui";
+
+export let loaded = false;
+
+export function set_up() {
+    loaded = true;
+    $("body").on("click", ".settings-unmute-user", function (e) {
+        const $row = $(this).closest("tr");
+        const user_id = Number.parseInt($row.attr("data-user-id"), 10);
+
+        e.stopImmediatePropagation();
```

Comment:
Why this rather than `.stopPropagation`?  I guess we do that in "Muted topics", but that's probably an accident.

Reply:
#18279 takes care of this in `settings/muted-topics`